packages/create-cloudflare/templates/analog/c3.ts

```diff
@@ -0,0 +1,268 @@
+import { crash, logRaw } from "@cloudflare/cli";
+import { brandColor, dim } from "@cloudflare/cli/colors";
+import { spinner } from "@cloudflare/cli/interactive";
+import { runFrameworkGenerator } from "frameworks/index";
+import { loadTemplateSnippets, transformFile } from "helpers/codemod";
+import { getLatestTypesEntrypoint } from "helpers/compatDate";
+import { readFile, writeFile } from "helpers/files";
+import { detectPackageManager } from "helpers/packageManagers";
+import { installPackages } from "helpers/packages";
+import * as recast from "recast";
+import type { TemplateConfig } from "../../src/templates";
+import type { C3Context } from "types";
+
+const { npm, name: pm } = detectPackageManager();
+
+const generate = async (ctx: C3Context) => {
+	await runFrameworkGenerator(ctx, [
+		ctx.project.name,
+		"--template",
+		"angular-v17",
+	]);
+
+	logRaw(""); // newline
+};
+
+const configure = async (ctx: C3Context) => {
+	// Fix hoisting issues with pnpm and yarn
+	if (pm === "pnpm" || pm === "yarn" || pm === "bun") {
+		const packages = [];
+		packages.push("nitropack");
+		packages.push("h3");
+		packages.push("@ngtools/webpack");
+		packages.push("@angular-devkit/build-angular");
+
+		await installPackages(packages, {
+			dev: true,
+			startText: `Installing ${packages.join(", ")}`,
+			doneText: `${brandColor("installed")} ${dim(`via \`${npm} install\``)}`,
+		});
+	}
+
+	updateViteConfig(ctx);
+	updateMainServer();
+	updateEnvTypes(ctx);
+};
+
+const updateEnvTypes = (ctx: C3Context) => {
+	const filepath = "env.d.ts";
+
+	const s = spinner();
+	s.start(`Updating ${filepath}`);
+
+	let file = readFile(filepath);
+
+	let typesEntrypoint = `@cloudflare/workers-types/`;
```

Comment:
```suggestion
	let typesEntrypoint = `@cloudflare/workers-types`;
```

Since:
![Screenshot 2024-04-10 at 22 02 54](https://github.com/cloudflare/workers-sdk/assets/61631103/8defb0ac-7712-4c1d-8db0-cac8a0bc3067)